examples/express/__tests__/router.test.ts

```diff
@@ -0,0 +1,15 @@
+import router from '../router';
+
+describe('Check if all endpoints are declarated', () => {
+  test('Should exists path /health', async () => {
+    const pathToBeTested = '/health';
+    const healthEndpoint = router.stack.find((endpoint: { route: { path: string; }; }) => endpoint.route.path === pathToBeTested);
+    expect(healthEndpoint).toBeDefined();
+  });
+  test('Should exists method GET on /health', async () => {
+    const pathToBeTested = '/health';
+    const healthMethod = router.stack.some((endpoint: { route: { path: string; methods: { get: any; }; }; }) => endpoint.route.path === pathToBeTested
```

Comment:
these tests seem to go too much into express implementation details, and express itself is already well tested
to me personally they do not look useful at all

there is a very popular library used for more thorough express testing 
https://www.npmjs.com/package/supertest

IMHO this does not look like something jest should add to its docs

Reply:
Yeah, the ideia is to create examples of how to test with express. 
The supertest lib tests express as a webserver, not the objetcs